README.md

```diff
@@ -1 +1,6 @@
-# chat-app
\ No newline at end of file
+# chat-app
+
```

Comment:
Please do mention about the variables to be added in .env file. Also user has to manually create .env file. Those information should be part of prerequisites in readme file

Reply:
ok (y)